.travis.yml

```diff
@@ -1,40 +1,52 @@
-language: python
+language: c
 
-python:
-    - 3.6
-
-sudo: false
+os:
+    - linux
 
 addons:
   apt:
     packages:
-    - graphviz
-    - texlive-latex-extra
-    - dvipng
+      - graphviz
+      - texlive-latex-extra
+      - dvipng
+
+stage: Comprehensive tests
+
+stages:
+   # Do the style check and a single test job, don't proceed if it fails
+   - name: Initial tests
+   # Test docs, astropy dev, and without optional dependencies
+   - name: Comprehensive tests
+   # These will only run when cron is opted in
+   - name: Cron tests
+     if: type = cron
 
-cache:
-  pip: true
-  #directories:
-  #  - $HOME/marxs-5.1.0
-  #  - $HOME/build/Chandra-MARX/marxs/marx/
 
 env:
     global:
         # The following versions are the 'default' for tests, unless
         # overridden underneath. They are defined here in order to save having
         # to repeat them for all configurations.
+        - PYTHON_VERSION=3.7
         - NUMPY_VERSION=stable
         - ASTROPY_VERSION=stable
+        - MAIN_CMD='python setup.py'
         - SETUP_CMD='test'
+        - EVENT_TYPE='pull_request push'
         - PIP_DEPENDENCIES='transforms3d'
         - CONDA_DEPENDENCIES='scipy cffi matplotlib jsonschema pyyaml'
+        - CONDA_DEPENDENCIES_DOC='sphinx-astropy'
```

Comment:
add  this as a pip dependency as this is a simple pure python package (although we package it up for conda and distribute on the astropy channel, you're not using that channel here)